boards/nordic/nrf54lv10dk/nrf54lv10dk_nrf54lv10a_cpuapp_ns.yaml

```diff
@@ -14,4 +14,4 @@ supported:
   - gpio
   - i2c
   - spi
-  - watchdog
\ No newline at end of file
+  - watchdog
```

Comment:
this is fixing up the previous commit, move the fixups parts to the previous commit